src/Components/Cases/constants.js

```diff
@@ -0,0 +1,25 @@
+export const STATUS_TO_DESCRIPTION = {
+  Unknown: 'Woops, we are not sure what is happening with this case',
+  'In progress': 'Dont worry, this case is currently being worked on',
+  Rejected: 'This case was rejected!',
+  Answered: 'This request has been answered',
+  'Has not been answered': 'This request has not been answered',
+  Withdrawn: 'This case was withdrawn',
+  Passed: 'Nice! This was passed!',
+};
+
+export const red = 'rgb(245,14,30)';
```

Comment:
It's usually not a good idea to use colour names in colour constants.

Instead they should represent the semantic use:

`export const warning = 'rgb(245, 14, 30)'`

This is good for two reasons:

1. If we decide some day that warnings should be orange; We won't end up with an orange warning label with the style 'red'
2. It's immediately clear which colour to use from the style guide; Is this a serious alert I'm showing the user? Then I'll use the `warning` constant

Pretty nitpicky comments I know. But it's something I've struggled with before :P

Reply:
thats true, I have changed this

Reply:
means we can also reuse this colors across the board later (and maybe abstract them into another file if needed)